daps-xslt/profiling/base-profile.xsl

```diff
@@ -218,7 +218,11 @@
    </xsl:attribute>
 </xsl:template>
 
-<!-- Remove any non-XInclude attributes -->
-<xsl:template match="xi:include/@*" mode="profile" />
+<!-- Remove any non-XInclude attributes
+
+    Issue #676: add priority="0" to overwrite default priority of 0.5 -> 0
+    This is needed to overcome a bug in libxslt >=1.1.35
```

Comment:
FWICT it's not a bug, but behavior according to the standard.

Reply:
You are right, I will change the wording.